src/stash/src/lib.rs

```diff
@@ -334,6 +334,13 @@ pub struct AppendBatch {
 }
 
 impl<K, V> StashCollection<K, V> {
+    /// Returns whether the collection is initialized. Collections that haven't been written to at
```

Comment:
I'm confused by this doc comment. Wouldn't a user want to call this function in order to determine if a collection has been written to or not?

Reply:
Yes, that's the intention. I was having trouble wording this. How's

> /// Returns whether the collection is initialized, i.e. has been written to at least once. Collections that haven't been written to at
> /// least once will panic if they're read from.

Reply:
I'm trying to describe what it means for a collection to be initialized and why it matters.

Reply:
They panic? Where? I thought `peek_timestamp_id` produces an error that bubbles up.

Reply:
Good point, I blindly copied that part of the comment from the controller's usage of `upper`. How about:

> ```
> /// Returns whether the collection is initialized, i.e. has been written to at least once. Collections that haven't been written to at
> /// least once cannot be read.
> ```

?

Reply:
SGTM